noir-projects/noir-protocol-circuits/bootstrap.sh

```diff
@@ -175,7 +175,7 @@ function build {
 }
 
 function test_cmds {
-  $NARGO test --list-tests --silence-warnings | sort | while read -r package test; do
+  $NARGO test --list-tests --silence-warnings --pedantic-solving | sort | while read -r package test; do
```

Comment:
```suggestion
  $NARGO test --list-tests --silence-warnings | sort | while read -r package test; do
```

We're just listing test functions here so no need to be pedantic.